data/formats-data.js

```diff
@@ -5700,7 +5707,7 @@ exports.BattleFormatsData = {
 		tier: "RU",
 	},
 	tyrunt: {
-		randomBattleMoves: ["stealthrock", "dragondance", "stoneedge", "dragonclaw", "earthquake", "icefang", "firefang"],
+		randomBattleMoves: ["dragondance", "stoneedge", "dragonclaw", "earthquake", "outrage", "rockslide"],
```

Comment:
keep fire fang

Reply:
why?

Reply:
it's one of those cases where firefang gets different coverage than eq, such as bronzor,and ferroseed.

Reply:
forgot tyrunt got strongjaw
or sturdy for that matter tbh